### ISSUE TITLE
Id of new work is not sequential

### ISSUE THREAD

**Issue by anusharanganathan** [culturesofknowledge/emlo-project#305]
Something odd is going on with the work numbering. I added these two records in quick succession, and the numbers of the works were not consecutive. As far as I know, I was the only person working in EMLO-Edit-new at that point. 

![error_nov24_6](https://github.com/user-attachments/assets/2331a1fa-8801-43ed-bf48-12ae4d36d5f4)

**Comment by anusharanganathan** [comment]
I also tried added  three letters and the Id for me was also not sequential. I don’t think anyone else added a letter  in the mean time [AR]

<img width="977" alt="Screenshot 2024-12-05 at 17 58 50" src="https://github.com/user-attachments/assets/d2db7486-a796-4f2b-a48a-31a0a3b1c0fe">

**Comment by anusharanganathan** [comment]
@philipkcl are you able to check this please

**Comment by philipkcl** [comment]
fixed but the change is too deep and complex could cause other issues

**Comment by anusharanganathan** [comment]
Hi @philipkcl, based on what you say, I will talk to the client and maybe it’s fine for it to not be sequential. Do you mind telling us why the Id is not sequential? There is no PR as yet, so I could not look at your code.

**Comment by philipkcl** [comment]
No, I have already implemented the fix in new branch. just ask them to report us if  any other new bugs created.

the reason is the seq_id will be increase every time when object created.

**Comment by anusharanganathan** [comment]
@MillieG711 please can you test this for me and see if new letters get IDs in sequence please

**Comment by MillieG711** [comment]
@anusharanganathan They are in sequence. Numbering seems to have started from 10000000 - when it gets deployed we'll need to have them starting from where the current IDs left off. I'll close the issue but make a note of this to check

**Comment by anusharanganathan** [comment]
@MillieG711 I have created a new issue for the ID to not start at 10000000, but to be one more than the largest work ID - #678
